drivers/hmis/app/models/hmis/hud/custom_data_element_definition.rb

```diff
@@ -21,28 +21,15 @@ class Hmis::Hud::CustomDataElementDefinition < Hmis::Hud::Base
     :json,
   ].freeze
 
-  SERVICE_OWNER_TYPES = ['Hmis::Hud::Service', 'Hmis::Hud::CustomService'].freeze
```

Comment:
Are these removals related to the main change in this PR or is it separate?

Reply:
It's unrelated, just removing unused code. Previously we thought we needed to tie the CDEDs to service types, but that didn't really make sense, since we are now using one CDED for multiple different service types.